src/bin/stratisd.rs

```diff
@@ -279,12 +262,25 @@ fn run() -> StratisResult<()> {
 }
 
 fn main() {
-    let error_code = match run() {
-        Ok(_) => 0,
-        Err(err) => {
-            print_err(&err);
-            1
-        }
-    };
-    exit(error_code);
+    let matches = App::new("stratis")
+        .version(VERSION)
+        .about("Stratis storage management")
+        .arg(Arg::with_name("debug")
+                 .long("debug")
+                 .help("Print additional output for debugging"))
+        .arg(Arg::with_name("sim")
+                 .long("sim")
+                 .help("Use simulator engine"))
+        .get_matches();
+
+    initialize_log(matches.is_present("debug"))
```

Comment:
I think that initialize() of log should occur after lock is gained. Otherwise there are two stratisd's both logging at the same time.